app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -210,6 +212,21 @@ class AwesomeBarView(
         return searchEngine
     }
 
+    /**
+     * Get a suggestions header if [currentEngineName] is the one of the default search engine.
+     *
+     * @param currentEngineName Name of the currently selected search engine.
+     */
+    private fun getSearchEngineSuggestionsHeader(currentEngineName: String?): String? {
+        val searchState = activity.components.core.store.state.search
```

Comment:
🔍 nit: since we're not using `searchState` directly, we can just get the `defaultSearchEngine` in one call.

Reply:
I think we currently have the two properties declared separately just to somehow be easier on the eye than a chain of 8 accessors which normally should be a big no-no.
Will combine the declarations.